app/assets/stylesheets/welcome.css

```
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;1,300&display=swap');
+
+.checkin-background {
+    align-items: center;
+    background-color: #a3eeff;
```

You can create a color library to create adding the colors you use in your style sheets.

```suggestion
    background-color: colors.color_name;
```

---

CSS doesn't have variables to hold data, and for this project, we skipping SASS.